staging/2022/benchmarks/json/benchmarks_cahps.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "measureId": "CAHPS_1",
+    "benchmarkYear": 2020,
+    "performanceYear": 2022,
+    "submissionMethod": "certifiedSurveyVendor",
+    "isToppedOut": false,
+    "isHighPriority": true,
+    "isToppedOutByProgram": false,
```

Comment:
Per Michelle, all CAHPS measures should have `isToppedOut` and `isToppedOutByProgram` to false; and all `isHighPriority` set to true (since measure 321 is high priority, and all these CAHPS measures are part of measure 321)